blueprints/initializer/files/__root__/initializers/__name__.js

```diff
@@ -2,7 +2,4 @@ export function initialize(/* application */) {
   // application.inject('route', 'foo', 'service:foo');
 }
 
-export default {
-  name: '<%= dasherizedModuleName %>',
-  initialize
-};
+export default { initialize };
```

Comment:
actually, the diff will be just as small/large as before unless you start to introduce trailing commas 😉